docs/spec/staking/state.md

```diff
@@ -2,46 +2,46 @@
 
 ### Pool
 
-The pool is a space for all dynamic global state of the Cosmos Hub.  It tracks
-information about the total amounts of Atoms in all states, moving Atom
-inflation information, etc.
+The pool dynamically tracks information about the total amounts of staking
+tokens and their states (bonded or loose). Note that `NotBondedTokens` _does_
+include both `unbonding` tokens, and fully `unbonded` tokens. 
 
  - Pool: `0x01 -> amino(pool)`
 
 ```golang
 type Pool struct {
-    LooseTokens         sdk.Int   // tokens not associated with any bonded validator
-    BondedTokens        sdk.Int   // reserve of bonded tokens
+    NotBondedTokens sdk.Int   // tokens not associated with any bonded validator
+    BondedTokens    sdk.Int   // reserve of bonded tokens
 }
 ```
 
 ### Params
 
 Params is global data structure that stores system parameters and defines
-overall functioning of the staking module. 
+overall functioning of the staking module. it is stored 
 
- - Params: `0x00 -> amino(params)`
+ - Params: `Paramsspace("staking") -> amino(params)`
```

Comment:
This is not informative to non-sdk devs IMO. Don't really know what is being conveyed here.

Reply:
Agreed - also this isn't the actual store key.

Reply:
I literally don't really comprehend how to adequately write the spec for the params store stuff at the moment.. I'm going to leave and open an issue for further thought.. @cwgoes if you have a good grasp on how this could be better depicted please push! New issue: https://github.com/cosmos/cosmos-sdk/issues/3326